roles/openshift_hosted/tasks/registry.yml

```diff
@@ -1,15 +1,18 @@
 ---
 - name: setup firewall
   include: firewall.yml
-  static: yes
+    l_openshift_hosted_firewall_enabled={{ r_openshift_hosted_registry_firewall_enabled }}
```

Comment:
nit: Since ``firewall`` is being shortened to ``fw`` in other variables it may be worth making this ``fw`` as well.

Reply:
I decided to keep this one 'firewall_enabled' because this variable is replacing the usage of an inventory variable that says "firewall_enabled" in it.